lib/bundler/cli/exec.rb

```diff
@@ -57,6 +57,7 @@ def kernel_load(file, *args)
       args.pop if args.last.is_a?(Hash)
       ARGV.replace(args)
       $0 = file
+      Process.setproctitle(process_title(file, *args))
```

Comment:
@yob I believe this is causing the CI failures as the `Process#setproctitle` method was first created/available in [Ruby `2.1.0`](http://ruby-doc.org/core-2.3.0/Process.html#method-c-setproctitle). As a side note, I have no idea why this is passing: https://travis-ci.org/bundler/bundler/jobs/131320827